blazor/mention/working-with-data.md

```diff
@@ -0,0 +1,187 @@
+* [DataManager.Adaptor](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DataManager.html#Syncfusion_Blazor_DataManager_Adaptor) - Defines the adaptor option. By default, the [ODataAdaptor](https://blazor.syncfusion.com/documentation/data/adaptors#odata-adaptor) is used for remote binding. The adaptor is responsible for processing responses and requests from or to the service endpoint. 
+* [Syncfusion.Blazor.Data](https://www.nuget.org/packages/Syncfusion.Blazor.Data/) package provides some predefined adaptors that are designed to interact with particular service endpoints.
+
+### OData v4 adaptor - Binding OData v4 service
+
+The [OData v4 Adaptor](https://blazor.syncfusion.com/documentation/data/adaptors#odatav4-adaptor) is an improved version of OData protocols, and the `DataManager` can also retrieve and consume OData v4 services.
+
+The following sample displays the first 6 contacts from `Customers` table of the `Northwind` Data Service.
+
+{% highlight razor %}
+
+{% include_relative code-snippet/odata-v4-service.razor %}
+
+{% endhighlight %}
+
+![Blazor Mention with OData v4 adaptor](./images/blazor-mention-ODatav4-adaptor.png)
+
+### Web API adaptor
+
+The [Web Api Adaptor](https://blazor.syncfusion.com/documentation/data/adaptors#web-api-adaptor) is used to interact with Web API created under OData standards. The `WebApiAdaptor` is extended from the [ODataAdaptor](https://blazor.syncfusion.com/documentation/data/adaptors#odata-adaptor). Hence to use the `WebApiAdaptor`, the endpoint should understand the OData formatted queries sent along with the request. 
+
+{% highlight razor %}
+
+{% include_relative code-snippet/web-api-adaptor.razor %}
+
+{% endhighlight %}
+
+![Blazor Mention with web API adaptor](./images/blazor-mention-web-api-adaptor.png)
+
+### Offline mode
+
+The `Offline` property of `DataManager` allows you to specify whether the data should be loaded from the server or from the local cache. If `Offline` is set to `true`, the `DataManager` will try to load data from the local cache first, and if that is not possible, it will try to load it from the server. This can be useful in situations where you want to minimise the number of requests to the server and improve its performance.
+
+The following example is for remote data binding and enabled `Offline` mode.
+
+{% highlight razor %}
+
+{% include_relative code-snippet/offline-mode.razor %}
+
+{% endhighlight %}
+
+![Blazor Mention with Offline mode](./images/blazor-mention-offline-mode.png)
+
+## Events
+
+### OnActionBegin event
+
+The `OnActionBegin` event is triggers whenever the Mention component starts to fetch data from a remote server using the `DataSource` or `DataManager` properties. The event is passed a `ActionEventArgs` object as a parameter, which contains information about the data fetch process.
```

Comment:
![image](https://user-images.githubusercontent.com/29145454/209297476-53e220ef-6a9d-401c-b01d-8706ddac6f09.png)


ActionEventArgs -- correct it

Reply:
Addressed